dali/test/python/test_pipeline.py

```diff
@@ -1091,60 +1091,67 @@ def iter_setup(self):
 
 
 class CachedPipeline(Pipeline):
-    def __init__(self, reader_type, batch_size, is_cached=False, seed=123456, skip_cached_images=False, num_shards=100000):
+    def __init__(self, reader_type, batch_size, is_cached=False, is_cached_batch_copy=True,  seed=123456, skip_cached_images=False, num_shards=100000):
```

Comment:
Better make `is_cached_batch_copy=False` and be explicit in the instantiation

Reply:
`is_cached_batch_copy` is the default option in nvJPEGDecoder. That's why I told Kyle to make it default here as well. This option should be only taken into consideration if `is_cached=True`